### ISSUE TITLE
[BUG]: users.update_user() ignores named parameters when building payload

### ISSUE THREAD

**Issue by calypso15** [mitchos/pyZscaler#110]
update_user() looks like this:

```
def update_user(
        self,
        user_id: str,
        name: str = None,
        email: str = None,
        department: dict = None,
        groups: list = None,
        **kwargs,
    ) -> Box:
        payload = convert_keys(self.get_user(user_id))

        # Add optional parameters to payload
        for key, value in kwargs.items():
            payload[snake_to_camel(key)] = value

        return self._put(f"users/{user_id}", json=payload)
```

The problem is that the named parameters (name, email, department, groups) are not part of kwargs. So it is impossible to update a user's name, email, department, or groups using this API call.

**Comment by mitchos** [comment]
Thanks for pointing this out @calypso15. I had written a test for this but looks like the test was broken and not working the way I thought.

I was assigning the `updated_user` dict to a variable (rather than use `dict.copy() `or `copy.deepcopy(dict)` and when I changed it, the original dict changes as well. Essentially causing the API request to pass no matter what.

I'll have a new release out in a few moments.

**Comment by mitchos** [comment]
Fixed in 1.1.1 release.

**Comment by calypso15** [comment]
Nice! Just pulled down 1.1.1 and confirmed it's working, thanks for the quick fix!
